cypress/integration/nameDetail.spec.js

```diff
@@ -40,12 +40,13 @@ function refreshAndCheckText(url, textOrArrayOfText) {
 describe('Name detail view', () => {
   it('can see list of top level domains from [root]', () => {
     cy.visit(`${NAME_ROOT}/[root]/subdomains`)
-    cy.queryByTestId('eth', { timeout: 10000 }).should('exist')
-    cy.queryByTestId('reverse', { timeout: 10000 }).should('exist')
+    cy.queryByTestId('eth', { timeout: 30000 }).should('exist')
```

Comment:
We have to wait for 30 seconds?

Reply:
I think the internet is slow on travis sometimes